app/serializers/atmosphere/action_log_serializer.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Action log serializer.
+#
+module Atmosphere
+  class ActionLogSerializer < ActiveModel::Serializer
+
+    attributes :id, :log_level, :message, :created_at
```

Comment:
I suspect you meant "appliance_id" - in fact it should be there but it's not. 
There is no such field as "action_id" - I think "id" works that way.